src/System/ref/Compat/System.cs

```diff
@@ -4995,14 +4995,14 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public CookieCollection() { }
         public int Count { get { return default(int); } }
         public bool IsReadOnly { get { return default(bool); } }
-        public bool IsSynchronized { get { return default(bool); } }
+        bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
+        object System.Collections.ICollection.SyncRoot { get { return default(object); } }
```

Comment:
I don't understand these changes.  The 'ref' file should contain only "public" or "protected" methods.  What does it mean to remove the "public" from these methods?  Does that mean that they are now "private"?  And if so, why list them at all in the 'ref' *.CS file.